src/main/java/seedu/duke/Parser.java

```diff
@@ -94,24 +95,34 @@ public static Command getAddMemberCommand(HashMap<String, String> params) throws
         return commandType;
     }
 
-    public static Command getDeadlineCommand(HashMap<String, String> params, boolean isProjectView)
+    public static void switchViewModes(boolean isProjectListView) throws DukeExceptions {
+        if (!isProjectListView) {
+            System.out.println("Switched to Project View!");
+            projectIndex = -1;
+        } else {
+            throw new DukeExceptions("Switch"); // REPLACED WITH EXCEPTION
+        }
+
+    }
```

Comment:
Hmm what is switchViewModes for? There's already home command to switch to home.

Reply:
Fixed!!